app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    @user = User.includes(:posts).find(params[:user_id])
     @posts = @user.posts
```

Comment:
- Hi, the requirements say **you should make sure that the N+1 problem is solved when fetching all posts and their comments for a user**. Here is an Example: in the posts#index actions you should use `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`